client/src/plus/deep-dives/series-card/index.scss

```diff
@@ -59,15 +58,15 @@
           font-weight: bold;
 
           &::before {
-            background-image: url("../../atoms/svg/active.svg");
+            background-image: url("./active.svg");
```

Comment:
I completely understand why you moved this here but, there is a good chance these might be reused on other pages as part of the redesign. I, therefore, opted to keep these all collocated in a single folder.

Reply:
That'll be excellent. There is only a matter of time where modules need to be shared and it's often better to *share* than to make duplicate copies. The day that comes, it's a battle we're going to need to face head-on. 
But what I prefer to do is to only move the "up the tree" as minimal as possible. 
For example, suppose the MDN Plus Bookmarks app needs that same `active.svg`, then you we can move it from `/src/plus/deep-dives/series-card/` to `/src/plus/` so at least it's all part of the `plus` namespace. 
Also, at some point, during that process, it might make sense to collect things into better names. For example, the `active.svg` can just sit there because the folder `/src/plus/deep-dives/series-card/` has, and always will, so few items. 
But moved up the tree it might make sense to put it into a `/src/plus/shared-icons/` folder or something like that.